app/styles/modules/_custom-rows.scss

```diff
@@ -14,6 +14,10 @@
     margin: 20px 0 15px 0;
   }
 
+  &.margin-top {
```

Comment:
Sorry, misunderstood your point, that works fine thanks. Now I'll go and look up what a combination selector is...